packages/terra-site/package.json

```diff
@@ -88,25 +88,25 @@
   "devDependencies": {
     "autoprefixer": "^6.7.7",
     "babel-cli": "^6.24.1",
-    "babel-loader": "^7.0.0",
+    "babel-loader": "^7.1.2",
     "babel-polyfill": "^6.23.0",
-    "clean-webpack-plugin": "^0.1.16",
-    "css-loader": "^0.28.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "file-loader": "^0.11.1",
+    "clean-webpack-plugin": "^0.1.17",
+    "css-loader": "^0.28.7",
+    "extract-text-webpack-plugin": "^3.0.1",
+    "file-loader": "^1.1.5",
     "gh-pages": "^0.12.0",
-    "html-webpack-plugin": "^2.28.0",
-    "json-loader": "^0.5.4",
+    "html-webpack-plugin": "^2.30.0",
+    "json-loader": "^0.5.7",
     "node-sass": "^4.5.2",
     "postcss-custom-properties": "^6.0.1",
-    "postcss-loader": "^1.3.3",
-    "postcss-rtl": "^0.5.10",
+    "postcss-loader": "^2.0.6",
+    "postcss-rtl": "^1.1.2",
     "raw-loader": "^0.5.1",
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
-    "sass-loader": "^6.0.3",
-    "style-loader": "^0.16.1",
-    "webpack": "^2.6.1",
-    "webpack-dev-server": "^2.4.5"
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.19.0",
+    "webpack": "^3.6.0",
```

Comment:
I'm not sure if we will be able to bump webpack versions until terra-toolkit's webpack version is bumped. I when I made changes in toolkit for nightwatch, I ran across webpack chunk issues in consumer when trying to run nightwatch tests. 

Do the nightwatch tests run without error?

Reply:
We'll see after travisCI runs.

Reply:
It appears there are nightwatch test failures. @mhemesath It would be good to sync with @emilyrohrbough and @ryanthemanuel on this.